libcodechecker/libhandlers/analyze.py

```diff
@@ -297,6 +296,46 @@ def add_arguments_to_parser(parser):
                                    "same translation unit without "
                                    "Cross-TU enabled.")
 
+    if host_check.is_statistics_capable():
+        stat_opts = parser.add_argument_group(
+            "statistics analysis arguments",
+            "These arguments are only available if the Clang Static Analyzer "
+            "supports Statistics-based analysis "
+            "(e.g. statisticsCollector.ReturnValueCheck,"
+            "statisticsCollector.SpecialReturnValue checkers are available).")
+        stat_opts.add_argument('--stats-collect', '--stats-collect',
+                               action='store',
+                               default=None,
+                               dest='stats_collect',
+                               help="Perform the first, 'collect' phase of "
+                                    "Statistical analysis. This phase"
```

Comment:
Please write a space before the last word of every such lines.
```bash
"Statistical analysis. This phase "
```
Plese check the other help messages and places too.